src/App.js

```diff
@@ -1,18 +1,53 @@
+import React, { useState } from "react";
 import "./App.scss";
 import Header from "./components/Header/Header";
 import Footer from "./components/Footer/Footer";
-import Icon from './components/Icon/Icon';
+import { TodayForecast } from "./components/TodayForecast/TodayForecast";
+import HourlyForecast from "./components/HourlyForecast/HourlyForecast";
+// import Icon from "./components/Icon/Icon";
+
 
 //configs
 const siteTitle = process.env.REACT_APP_SITE_TITLE ?? "CYF Weather";
 
 function App() {
+  const [data, setData] = useState([]);
+
+  function getNewLocation(city) {
+    fetch(
+      `https://api.openweathermap.org/data/2.5/forecast?q=${city}&units=metric&appid=7e5286e19236105eab5bba5240bf165e`
+    )
+      .then((response) => response.json())
+      .then((data) => {
+        setData(data);
+      });
```

Comment:
Every fetch should have a .catch clause at the end in case it errors, otherwise it will fail silently

Reply:
Yes sure. Done!